install_config/aggregate_logging.adoc

```diff
@@ -3847,3 +3850,230 @@ cluster. Edit your non-cluster logging service (for example, `logging-es`,
+
+===  All Components
+
+*Deployment fails, RCs scaled to 0*
+
+When a deployment is performed, if it does not successfully bring up an
+instance before a ten-minute timeout, it will be considered failed and
+scaled down to zero instances. `oc get pods` will show a deployer pod
+with a non-zero exit code, and no deployed pods, e.g.:
+
+----
+NAME                           READY     STATUS             RESTARTS   AGE
+logging-es-2e7ut0iq-1-deploy   1/1       ExitCode:255       0          1m
+----
+
+(In this example, the deployer pod name for an Elasticsearch deployment is shown;
+this is from ReplicationController `logging-es-2e7ut0iq-1` which is a deployment
+of DeploymentConfig `logging-es-2e7ut0iq`.)
+
+Deployment failure can happen for a number of transitory reasons, such as
+the image pull taking too long, or nodes being unresponsive. Examine the
+deployer pod logs for possible reasons; but often you can redeploy:
+
+----
+$ oc deploy --latest logging-es-2e7ut0iq
+----
+
+Or you may be able to scale up the existing deployment:
+
+----
+$ oc scale --replicas=1 logging-es-2e7ut0iq-1
+----
+
+If the problem persists, examine pod, events, and systemd unit
+logs to determine the source of the problem.
+
+*Image pull fails*
+
+If you specify an `openshift_logging_image_prefix` that results in images being defined that don't exist,
+you will receive a corresponding error message:
+
+----
+NAME                     READY     STATUS                                                                                       RESTARTS   AGE
+logging-fluentd-1ub9k    0/1       Error: image registry.access.redhat.com:5000/openshift3logging-fluentd:latest not found      0          1m
+----
+
+In this example, for the intended image name
+`registry.access.redhat.com:5000/openshift3/logging-fluentd:latest`
+the `openshift_logging_image_prefix` needed a trailing `/`.
+
+Update the inventory file and follow the `openshift-ansible` instructions
+to re-run the `openshift_logging` role.
+
+*Can't resolve kubernetes.default.svc.cluster.local*
+
+This internal alias for the master should be resolvable by the included
+DNS server on the master. Depending on your platform, you can run the `dig` command (perhaps in a container) against the master to
+check whether this is the case:
+
+----
+$ dig kubernetes.default.svc.cluster.local @localhost
+[...]
+;; QUESTION SECTION:
+;kubernetes.default.svc.cluster.local. IN A
+
+;; ANSWER SECTION:
+kubernetes.default.svc.cluster.local. 30 IN A   172.30.0.1
+----
+
+Older versions of OKD did not automatically define this internal
+alias for the master. You may need to upgrade your cluster in order to
+use aggregated logging. If your cluster is up to date, there may be
+a problem with your pods reaching the SkyDNS resolver at the master,
+or it could have been blocked from running. You must resolve this
+problem before deploying again.
+
```

Comment:
@jcantrill Does this paragraph apply to OKD only, or the entire troubleshooting item?
Or, should this reference to OKD be removed for the OpenShift docs?

Reply:
You could probably remove reference to 'OKD' and simply call it 'cluster logging' or some variant everywhere